desktop-exporter/app/components/waterfall-view/waterfall-row.tsx

```diff
@@ -37,7 +37,7 @@ export function WaterfallRow({ index, style, data }: WaterfallRowProps) {
   // Set the background colour to make the list striped.
   let backgroundColour =
     index % 2 ? "" : useColorModeValue("gray.50", "gray.700");
```

Comment:
There is a potential issue with selectively calling the hook here similar to what we saw in `SidebarRow`